watertap/unit_models/zero_order/anaerobic_digestion_oxidation_zo.py

```diff
@@ -0,0 +1,41 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of an anaerobic digestion oxidation unit.
+operation.
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_sido, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("AnaerobicDigestionOxidationZO")
+class AnaerobicDigestionOxidationZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for an Anaerobic Digestion Oxidation unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "anaerobic_digestion_oxidation"
```

Comment:
I am wondering why this was originally named "anaerobic_digestion_oxidation" instead of just "anaerobic_digestion" in WaterTAP3. @kurbansitterley any thoughts? I'd prefer this to just be "anaerobic_digestion" if we don't need the "oxidation" part.